neuro_san/test/driver/assert_capture.py

```diff
@@ -254,3 +254,33 @@ def assertLessEqual(self, first: Any, second: Any, msg: str = None):
             self.basis.assertLessEqual(first, second, msg)
         except AssertionError as exception:
             self.asserts.append(exception)
+
+    # pylint: disable=invalid-name
+    def assertGist(self, gist: bool, acceptance_criteria: str, text_sample: str, msg: str = None):
```

Comment:
Add nominal assert{Not}Gist() implementations to this guy just like all the rest.
Recall that this AssertCapture class collects failures and defers on reporting them in case the sucess_ratio is < 1.0

Reply:
Why `assertGist` and not `assert_gist`? You have to use ` pylint: disable=invalid-name` all over the place but I don't see why these methods can't be regular Python functions that follow the standard conventions.

Reply:
It's how all the other TestCase.assert* are put together. Weird to have a style change mid-class.
Decent point though.